clang/test/CodeGenSYCL/inline_asm.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -emit-llvm -x c++ %s -o - | FileCheck %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-enable-optimizations -triple spir64-unknown-unknown-sycldevice -emit-llvm -x c++ %s -o - | FileCheck %s
```

Comment:
I don't get this change. If you want to test your new option, add specific test for it. I don't see why you adding this option to unrelated test and check noting.

Reply:
It's explained in the commit message.
"[SYCL] Update the test checking optimized LLVM IR"

Reply:
Ok, I see. No more questions about the change.
We added a test checking optimized IR. Does such test makes sense if we don't enable optimizations when compiling SYCL application?

Reply:
+@premanandrao,

It still makes sense because it has a few checks matching asm, which are relevant as target independent optimizations do not optimize inline asm.

Reply:
Although I agree that it would be better to re-write the checks.